src/ApiPort.VisualStudio/AnalyzeMenu.cs

```diff
@@ -171,5 +184,23 @@ private ICollection<Project> GetSelectedProjects()
                 .Distinct()
                 .ToList();
         }
+
+        private ICollection<Project> GetTransitiveReferences(IEnumerable<Project> projects, HashSet<Project> expandedProjects = null)
```

Comment:
Good catch - I must have changed this without trying it out. I was checking the wrong variable! I'm definitely switching it back to what I had